### ISSUE TITLE
Basic AI sometimes wants to play wild/wild

### ISSUE THREAD

**Issue by kingofzeal** [SignetStudios/slackbot-uno#17]
Should never attempt to play a wild without specifying a non-wild color.

**Comment by kingofzeal** [comment]
Seems to occur when following a wild being played from the previous player.

The following is relevant from the logs when this happened:

```
[   2016-11-06T16:12:54Z    ]   basicAI selected blue wild to play.
[   2016-11-06T16:12:54Z    ]   Ending turn for basicAI
[   2016-11-06T16:12:54Z    ]   [ { color: 'green', value: '3' },
[   2016-11-06T16:12:54Z    ]   { color: 'green', value: '6' },
[   2016-11-06T16:12:54Z    ]   { color: 'yellow', value: '8' },
[   2016-11-06T16:12:54Z    ]   { color: 'green', value: '1' },
[   2016-11-06T16:12:54Z    ]   { color: 'yellow', value: '5' },
[   2016-11-06T16:12:54Z    ]   { color: 'wild', value: 'wild' } ]
[   2016-11-06T16:12:54Z    ]   { color: 'blue', value: 'wild' }
[   2016-11-06T16:12:54Z    ]   basicAI2 selected wild wild to play.
```

In this case, it didn't have a blue to follow with, and decided to play its _own_ wild, but somehow got stuck on 'wild' being the color. I believe this is because it's trying to follow with the _value_ of the card, and thus is bypassing all the wild logic.

**Comment by kingofzeal** [comment]
Fixed with 51bc3a5c43d8c58cd7223b4cabf822d26a5754d3
